app/Http/Controllers/Admin/CustomToursLandingPageItemController.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use A17\Twill\Http\Controllers\Admin\ModuleController;
+
+class CustomToursLandingPageItemController extends ModuleController
+{
+    protected $moduleName = 'customToursLandingPageItems';
```

Comment:
I wonder if the module could be called `customToursItems`? So it's less verbose?

Reply:
Good idea, I've updated 👍